core/chain-configs/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 mod client_config;
 mod genesis_config;
+pub mod genesis_validate;
 
 pub use client_config::{ClientConfig, LogSummaryStyle, TEST_STATE_SYNC_TIMEOUT};
 pub use genesis_config::{
-    Genesis, GenesisConfig, GenesisRecords, ProtocolConfig, ProtocolConfigView,
+    get_initial_supply, Genesis, GenesisConfig, GenesisRecords, ProtocolConfig, ProtocolConfigView,
 };
+#[cfg(feature = "protocol_feature_evm")]
+pub use genesis_config::{BETANET_EVM_CHAIN_ID, MAINNET_EVM_CHAIN_ID, TESTNET_EVM_CHAIN_ID};
```

Comment:
```suggestion
```

we removed `evm` feature from master branch

Reply:
Done